concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1338,6 +1338,73 @@ jobs:
                   ./paas-cf/concourse/scripts/set_quotas_from_manifest.rb cf-manifest/cf-manifest.yml
 
       - aggregate:
+        - task: deploy-test-application
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+            params:
+              TEST_HEAVY_LOAD: {{test_heavy_load}}
+            inputs:
+              - name: paas-cf
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  if [ "${TEST_HEAVY_LOAD:-}" -eq "false" ] ; then
+                    echo Heavy load test has been disabled. Skipping...
+                    exit 0
+                  fi
+
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  cf create-org heavy-testers
+                  cf set-quota heavy-testers medium
+                  cf create-space test-application -o heavy-testers
+                  cf target -o heavy-testers -s test-application
+
+                  cd paas-cf/platform-tests/example-apps/healthcheck
+                  cf push test-application
```

Comment:
I think we should give this app a more descriptive name. How about simulated-load or something like that.